src/Amadeus/Client/RequestOptions/Service/StandaloneCatalogue/Passenger.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * amadeus-ws-client
+ *
+ * Copyright 2015 Amadeus Benelux NV
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @package Amadeus
+ * @license https://opensource.org/licenses/Apache-2.0 Apache 2.0
+ */
+
+namespace Amadeus\Client\RequestOptions\Service\StandaloneCatalogue;
+
+use Amadeus\Client\LoadParamsFromArray;
+
+/**
+ * Passenger
+ *
+ * @package Amadeus\Client\RequestOptions\Fare\InformativePricing
```

Comment:
If this is an exact copy of `Amadeus\Client\RequestOptions\Fare\InformativePricing`, then you should just `use`  the existing class instead of copying it. I want as little as possible duplicate code in the library.

Reply:
Hi, this is not exact copy of Amadeus\Client\RequestOptions\Fare\InformativePricing.
I made some change that is require of this request. Instead if Tatoo that is array data type, we need reference that is index for each passenger.

Reply:
Okay, in that case it's perhaps better to `extend` from the InformativePricing class,